types/spotify-api/index.d.ts

```diff
@@ -468,10 +468,18 @@ declare namespace SpotifyApi {
      * Get user's saved shows
      * 
      * GET /v1/me/shows
-     * https://developer.spotify.com/documentation/web-api/reference/library/get-users-saved-shows/
+     * https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-users-saved-shows/
```

Comment:
Old link works, new one doesn't. Why are you changing this?

Reply:
I've updated it based on the documentation, it wasn't working because of the slash just like the other one. Good catch!